canvas_modules/common-canvas/src/common-properties/panels/summary/summary.jsx

```diff
@@ -265,13 +265,13 @@ class SummaryPanel extends React.Component {
 	render() {
 		const icon = this._getSummaryIconState();
 		const link = (<div className="properties-summary-link-container">
-			<IconButton
-				className="properties-summary-link-button"
-				icon={<Icon type={CARBON_ICONS.ADD} />}
+			<Link
```

Comment:
After clicking on one of the summary links, all the links became "visited" across different parameterDefs. Should we disable `visited` or ensure each link is unique? 
<img width="312" alt="image" src="https://user-images.githubusercontent.com/1628100/82927461-9a5b9e00-9f35-11ea-9a3a-3d767f9dfe40.png">